conans/test/unittests/model/options_test.py

```diff
@@ -206,6 +206,21 @@ def test_dumps(self):
         text = self.sut.dumps()
         assert text == ""
 
+    def test_boolean_none(self):
+        options = Options({"static": [None, "None", 1, 2]})
```

Comment:
Looks good and your comment in the linked issue nicely explains how this works now.

Would maybe adding some asserts like `assert not options.static` be useful?

This might just be me being clueless yet as to how things work (even after reading some of the code) but I had to double check that `assert not options.static` would work as expected locally, and that for `state = None` the assert would pass (which it did and I expected it it), and that `static = "None"` would not pass the assert, which it did pass so I was not expecting that!

Reply:
The case ``assert not options.static`` is already sufficiently covered in the test above, I was only covering here the pending bug of confusing Python ``None`` with a string ``"None"``, which was really more confusing.